### PR TITLE
Pb 6239 delete pod

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
Added torpedo utils for :
- DeletePod
- GetPxBackupNamespace
Modified ScheduleAppsAndValidate to accept app list instead if just one app

**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PB-6239

**Special notes for your reviewer**:

Verified this on cypress
- passing only label
- passing only pod list
- passing both (label takes priority)

### CODE REVIEW

File: apiServer/taas/utils/torpedoutils.go [hunk 1]
```diff
@@ -177,44 +178,56 @@ func CollectSupport(c *gin.Context) {
...
+		NamespaceSuffix string   `json:"nsSuffix"`
+		AppList         []string `json:"appList"`
+	}
 	var errors []error
+	errStrings := make([]string, 0)
 	errChan := make(chan error, 100)
 	if !checkTorpedoInit(c) {
 		c.JSON(http.StatusInternalServerError, gin.H{
-			"error": fmt.Errorf("Error happened while doing InitInstance()"),
+			"error": fmt.Errorf("error happened while doing InitInstance()"),
```
Comment (mkoppal-px) [inline apiServer/taas/utils/torpedoutils.go:191]: nit: change it to `error during InitInstance`

File: apiServer/taas/utils/torpedoutils.go [hunk 2]
```diff
@@ -500,3 +513,66 @@ func UpgradeStork(c *gin.Context) {
...
 }
+
+// DeletePod deletes the pods with given label
+func DeletePod(c *gin.Context) {
+	log.Infof("Deleting pods with given label")
+	var deletePodRequest struct {
+		Namespace   string            `json:"namespace"`
+		Label       map[string]string `json:"label"`
+		IgnoreLabel bool              `json:"ignoreLabel"`
+	}
```
Comment (mkoppal-px) [inline apiServer/taas/utils/torpedoutils.go:524]: We should accept pod name as well here

Comment (snigdha-px) [reply apiServer/taas/utils/torpedoutils.go]: done

File: apiServer/taas/utils/torpedoutils.go (hunk 1 above)
Comment (snigdha-px) [reply apiServer/taas/utils/torpedoutils.go]: done

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by gvavinash — APPROVED** [review]

**Pull request merged**